src/app/loans/loans-view/loan-account-actions/undo-approval/undo-approval.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+
+  <mat-card-content>
+
+    <mat-form-field fxLayoutAlign="center">
+      <mat-label>Note</mat-label>
+      <textarea matInput [formControl]="note"></textarea>
+    </mat-form-field>
+
+    <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+      <button type="button" mat-raised-button [routerLink]="['../']">Cancel</button>
+      <button mat-raised-button color="primary" (click)="submit()">Submit</button>
```

Comment:
```suggestion
      <button mat-raised-button color="primary" (click)="submit()" *mifosxHasPermission="'APPROVALUNDO_LOAN'" >Submit</button>
```
Please add the permission directive as well.